notepad.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE notes (
+ID INT PRIMARY KEY AUTO_INCREMENT , 
+title varchar(255) UNIQUE NOT NULL, 
+textbody text
```

Comment:
Use 'varchar(MAX)' instead of 'text'. The text datatype is deprecated and should not be used for new development work.